conda-requirements.txt

```diff
@@ -2,7 +2,7 @@
 # conda create -n <name> --file conda-requirements.txt
 
 # Mandatory dependencies
-biggus>=0.8
+biggus=0.11.0
```

Comment:
You've fixed the version of Biggus to v0.11.0 here, but it should be v0.11.0 _or greater_, as this is a minimum version requirement.